<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>980</width>
    <height>580</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Rahbin</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../.designer/build-Rahbin_GUI_v2-Desktop_Qt_5_7_0_GCC_64bit-Debug/rahbin.png</normaloff>../../.designer/build-Rahbin_GUI_v2-Desktop_Qt_5_7_0_GCC_64bit-Debug/rahbin.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="setup_tab">
       <attribute name="title">
        <string>Setup</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="1">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Maximum</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="install_btn">
                <property name="font">
                 <font>
                  <family>DejaVu Sans Mono</family>
                  <pointsize>11</pointsize>
                  <weight>75</weight>
                  <italic>true</italic>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Install</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="run_btn">
                <property name="font">
                 <font>
                  <family>DejaVu Sans Mono</family>
                  <pointsize>11</pointsize>
                  <weight>75</weight>
                  <italic>true</italic>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Run</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="stop_btn">
                <property name="font">
                 <font>
                  <family>DejaVu Sans Mono</family>
                  <pointsize>11</pointsize>
                  <weight>75</weight>
                  <italic>true</italic>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Stop</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="reboot_btn">
                <property name="font">
                 <font>
                  <family>DejaVu Sans Mono</family>
                  <pointsize>11</pointsize>
                  <weight>75</weight>
                  <italic>true</italic>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>reboot</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Preferred</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QGridLayout" name="gridLayout_6">
              <property name="sizeConstraint">
               <enum>QLayout::SetNoConstraint</enum>
              </property>
              <property name="bottomMargin">
               <number>10</number>
              </property>
              <property name="spacing">
               <number>5</number>
              </property>
              <item row="2" column="1">
               <widget class="QGroupBox" name="groupBox">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="font">
                 <font>
                  <family>Century Schoolbook L</family>
                  <pointsize>20</pointsize>
                  <weight>75</weight>
                  <italic>true</italic>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="title">
                 <string>Installation options</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <layout class="QGridLayout" name="gridLayout_2">
                 <item row="6" column="0">
                  <widget class="QCheckBox" name="show_ttc_chb">
                   <property name="font">
                    <font>
                     <family>Century Schoolbook L</family>
                     <pointsize>14</pointsize>
                     <weight>75</weight>
                     <italic>true</italic>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="text">
                    <string>Show TTC</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="1">
                  <widget class="QCheckBox" name="save_ldw_chb">
                   <property name="font">
                    <font>
                     <family>Century Schoolbook L</family>
                     <pointsize>14</pointsize>
                     <weight>75</weight>
                     <italic>true</italic>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="text">
                    <string>Save LDW</string>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="1">
                  <widget class="QCheckBox" name="tracker_chb">
                   <property name="font">
                    <font>
                     <family>Century Schoolbook L</family>
                     <pointsize>14</pointsize>
                     <weight>75</weight>
                     <italic>true</italic>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="text">
                    <string>Tracker</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="0">
                  <widget class="QCheckBox" name="save_fcw_chb">
                   <property name="font">
                    <font>
                     <family>Century Schoolbook L</family>
                     <pointsize>14</pointsize>
                     <weight>75</weight>
                     <italic>true</italic>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="text">
                    <string>Save FCW</string>
                   </property>
                   <property name="checked">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item row="7" column="0">
                  <widget class="QCheckBox" name="display_chb">
                   <property name="font">
                    <font>
                     <family>Century Schoolbook L</family>
                     <pointsize>14</pointsize>
                     <weight>75</weight>
                     <italic>true</italic>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="text">
                    <string>Display on</string>
                   </property>
                  </widget>
                 </item>
                 <item row="5" column="0">
                  <widget class="QCheckBox" name="roi_chb">
                   <property name="font">
                    <font>
                     <family>Century Schoolbook L</family>
                     <pointsize>14</pointsize>
                     <weight>75</weight>
                     <italic>true</italic>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="text">
                    <string>Show ROI</string>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="0">
                  <widget class="QCheckBox" name="alaram_chb">
                   <property name="font">
                    <font>
                     <family>Century Schoolbook L</family>
                     <pointsize>14</pointsize>
                     <weight>75</weight>
                     <italic>true</italic>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="text">
                    <string>Alaram</string>
                   </property>
                   <property name="checked">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="0">
                  <widget class="QCheckBox" name="FCW_chb">
                   <property name="font">
                    <font>
                     <family>Century Schoolbook L</family>
                     <pointsize>14</pointsize>
                     <weight>75</weight>
                     <italic>true</italic>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="text">
                    <string>FCW</string>
                   </property>
                   <property name="checked">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QCheckBox" name="ldw_chb">
                   <property name="font">
                    <font>
                     <family>Century Schoolbook L</family>
                     <pointsize>14</pointsize>
                     <weight>75</weight>
                     <italic>true</italic>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="text">
                    <string>LDW</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QCheckBox" name="save_in_chb">
                   <property name="font">
                    <font>
                     <family>Century Schoolbook L</family>
                     <pointsize>14</pointsize>
                     <weight>75</weight>
                     <italic>true</italic>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="text">
                    <string>Save Input</string>
                   </property>
                   <property name="checked">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QCheckBox" name="save_out_chb">
                   <property name="font">
                    <font>
                     <family>Century Schoolbook L</family>
                     <pointsize>14</pointsize>
                     <weight>75</weight>
                     <italic>true</italic>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="text">
                    <string>Save Output</string>
                   </property>
                  </widget>
                 </item>
                 <item row="5" column="1">
                  <widget class="QCheckBox" name="show_detect_chb">
                   <property name="font">
                    <font>
                     <family>Century Schoolbook L</family>
                     <pointsize>14</pointsize>
                     <weight>75</weight>
                     <italic>true</italic>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="text">
                    <string>Show Detectoins</string>
                   </property>
                  </widget>
                 </item>
                 <item row="6" column="1">
                  <widget class="QCheckBox" name="show_dist_chb">
                   <property name="font">
                    <font>
                     <family>Century Schoolbook L</family>
                     <pointsize>14</pointsize>
                     <weight>75</weight>
                     <italic>true</italic>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="text">
                    <string>Show dist</string>
                   </property>
                  </widget>
                 </item>
                 <item row="7" column="1">
                  <widget class="QCheckBox" name="regressor_chb">
                   <property name="font">
                    <font>
                     <pointsize>14</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>Regressor</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="username_text">
                <property name="text">
                 <string>pi</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>User</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="user_IP_text">
                <property name="text">
                 <string>10.42.0</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>IP</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QLabel" name="label_3">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="font">
                 <font>
                  <family>Century Schoolbook L</family>
                  <pointsize>20</pointsize>
                  <weight>75</weight>
                  <italic>true</italic>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Calibration</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item row="5" column="1">
               <widget class="QPushButton" name="calib4p_btn">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>4 Points</string>
                </property>
               </widget>
              </item>
              <item row="6" column="1">
               <widget class="QPushButton" name="calibHorizon_btn">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Horizon</string>
                </property>
               </widget>
              </item>
              <item row="5" column="0">
               <widget class="QPushButton" name="calib_4p_ok_btn">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>OK</string>
                </property>
               </widget>
              </item>
              <item row="6" column="0">
               <widget class="QPushButton" name="calib_horizon_ok_btn">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>OK</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="status">
      <property name="text">
       <string>status : ready</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QTabWidget" name="stream_tab">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="stream">
       <attribute name="title">
        <string>Stream</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="stream_label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="textFormat">
           <enum>Qt::PlainText</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
